packages/cli/test/functional/test_site/testList.md

```diff
@@ -0,0 +1,19 @@
+* Item 1 <br>
+  Item 1 line 2  {icon="fas-file-code" i-class="text-warning"}
+* Item 2 
+  Item 2 line 1 continue 
+  * Item 2.1 {icon="fas-code-branch" i-class="text-success"}
+  * Item 2.2 
+  * Item 2.3 {i-class="text-danger"}
+    * Item 2.3.1 {icon="fas-code-branch" i-class="text-success"}
+* Item 3
+  * Item 3.1 
+  * Item 3.2 {i-class="text-primary"}
+  * Item 3.3 
+    * Item 3.3.1 
+      * Item 3.3.1.1 
+    * Item 3.3.2 {icon="fas-code-branch" i-class="text-success"}
+  * Item 3.4 {icon="fas-file-code" i-class="text-warning"} 
+  * Item 3.4
+* item 4 {icon="fas-question-circle" i-class="badge rounded-pill mb-1 bg-success text-white"}
+  * item 4.1 {icon="./images/deer.png" i-width="50px"}
```

Comment:
A few more cases we can verify:
- default list + nested list without anything
- no `icon` in first item of the list but in the second does nothing
- 1 item list + nested list

Reply:
missing `1 item list + nested list`. e.g.

```
- Only 1 item

- Only 1 item
   - Only 1 item
```